.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: Go
```

Comment:
Not sure why this doesn't kick in with the current PR. Will verify when we have our next PR.